ApsimNG/Utility/FindAndReplaceForm.cs

```diff
@@ -69,17 +72,37 @@ private void Window1_Destroyed(object sender, EventArgs e)
 
         public void Destroy()
         {
+#if NETCOREAPP
+            window1.Dispose();
+#else
             window1.Destroy();
+#endif
```

Comment:
This conditional code could be replaced with a call to `window1.Cleanup()`, which is an extension method in the `UserInterface.Extensions` namespace:

```suggestion
            window1.Cleanup();
```

Reply:
Ah, right. I thought there was something like that.